Formula/fakeroot.rb

```diff
@@ -1,75 +1,41 @@
 class Fakeroot < Formula
   desc "Provide a fake root environment"
   homepage "https://tracker.debian.org/pkg/fakeroot"
-  url "https://deb.debian.org/debian/pool/main/f/fakeroot/fakeroot_1.24.orig.tar.gz"
-  sha256 "2e045b3160370b8ab4d44d1f8d267e5d1d555f1bb522d650e7167b09477266ed"
-  license "GPL-3.0"
+  url "https://deb.debian.org/debian/pool/main/f/fakeroot/fakeroot_1.29.orig.tar.gz"
+  sha256 "8fbbafb780c9173e3ace4a04afbc1d900f337f3216883939f5c7db3431be7c20"
+  license "GPL-3.0-or-later"
 
   bottle do
     sha256 cellar: :any, catalina:    "c72ae187158b6cce73311fee527ba8bf8d2f0e18340bd66eef57b50b3d45c275"
     sha256 cellar: :any, mojave:      "6c23e4c601af569c2de802cac685de5d18e6ebafcb53e6c53107aa3feb3d1527"
     sha256 cellar: :any, high_sierra: "df9be392f3579464893be013744b5aa40a7e4e91e01155bd1547e4104d381640"
   end
 
-  # Does not build. Mac support looks abandoned since 2013
-  # Initial mac support: https://github.com/mackyle/fakeroot
-  # https://salsa.debian.org/clint/fakeroot/-/blob/master/README_MACOSX.txt
-  disable! date: "2022-03-28", because: :does_not_build
+  # Needed to apply patches below. Remove when no longer needed.
+  depends_on "autoconf" => :build
+  depends_on "automake" => :build
+  depends_on "libtool" => :build
 
-  on_linux do
-    depends_on "libcap"
-  end
```

Comment:
An earlier run showed no linkage. Arch and Debian also do not have a `libcap` dependency.

Reply:
Arch does have an indirect dependency to `libcap-ng` via `util-linux`

Debian has build dependency on `libcap` https://packages.debian.org/source/sid/fakeroot

Removing it may be causing current failure:
```
libtool: compile:  gcc-5 -DHAVE_CONFIG_H -I. -g -O2 -c libfakeroot.c  -fPIC -DPIC -o .libs/libfakeroot.o
  In file included from libfakeroot.c:271:0:
  wrapped.h:127:27: error: unknown type name ‘cap_user_header_t’
   extern int (*next_capset)(cap_user_header_t hdrp, const cap_user_data_t datap);
```